crates/db_views/community/src/lib.rs

```diff
@@ -43,3 +49,17 @@ pub struct CommunityView {
   )]
   pub post_tags: TagsView,
 }
+
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct MultiCommunityView {
+  pub multi: MultiCommunity,
+  pub entries: Vec<CommunityId>,
+}
```

Comment:
Delete. Fetching the multi_community, and the list of communities it contains, need to be different fetches anyway.